ceremony/2022-10-03/staged/root.json

```diff
@@ -156,6 +156,46 @@
 			"keyid": "2f64fb5eac0cf94dd39bb45308b98920055e9a0d8e012a7220787834c60aef97",
 			"sig": ""
 		},
+		{
+			"keyid": "f40f32044071a9365505da3d1e3be6561f6f22d0e60cf51df783999f6c3429cb",
+			"sig": ""
+		},
```

Comment:
Where are these empty entries coming from? The other PRs do not have them, as far as I have seen

Reply:
As noted up-thread https://github.com/sigstore/root-signing/pull/439#pullrequestreview-1131295912 these are due to a bug in metadata generation code that was fixed in https://github.com/sigstore/root-signing/pull/435.

Reply:
Yes: thank you for the close look @flxw!!

I assume Dan's still busy at the workshop: so therefore we have four options:
1. Do not merge this one, and keep the threshold of 3 sigs created by @joshuagl @mnm678 @bobcallaway 
2. Merge this one, and snapshot: snapshotting DOES clear empty placeholder signatures (https://github.com/sigstore/root-signing/blob/dfcac5eaf9f204da67d271207bd7af7ff8624263/cmd/tuf/app/snapshot.go#L72). The reason why I had to make that fix though is that others who may sign on top of these extra empty entries would get messed up. It's "OK" if this is the final PR going in.
3. Manually remove these empty entries with a secondary PR or a commit here. Downside is that we would mess with the metadata outside the code-generated outputs.
4. I can stage the rest of the changes, and we can wait and see if Dan has time to rebase.